### PR TITLE
[CI-RUNNER] Output the build log directly to the STDOUT

### PR DESCRIPTION
**Pull request by marpme**
For most CI's the constant output while running a task is necessary to keep the CI alive.
So whenever a CI is running gitian it should check the CI env to enable direct logs to the CI STDOUT.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Couldn't it log to both, so that the file is always created and exists?

**Comment by devrandom** [comment]
agreed with @MarcoFalke

**Comment by marpme** [comment]
Okay, I will then basically build in the following: `... 2>&1 | tee var/build.log`

File: bin/gbuild [hunk 1]
```diff
@@ -159,9 +159,13 @@ EOF" if build_desc["sudo"] and @options[:allow_sudo]
     script.puts "cd build"
     script.puts build_desc["script"]
   end
-
+
```
Comment (MarcoFalke) [inline bin/gbuild]: Why this change?

Comment (marpme) [reply bin/gbuild]: Well the moment, you don't have a real code editor handling the whitespace situation for you 💃 
Good find tho

**Comment by marpme** [comment]
Anything else incorrect/missing for you?
